packages/front-end/styles/theme-config.css

```diff
@@ -14,4 +14,9 @@
 .light,
 .light-theme {
   --color-page-background: #f5f7fa;
+  --color-background: #faf8ff;
+}
+.dark,
+.dark-theme {
+  --color-background: #10172e;
```

Comment:
question: Do we need to define these new values or should we use `--color-panel-solid` which is provided by Radix?

Reply:
I think we need to because these are custom colors we came up with for our design system. I’ll double check that they’re not the same

Reply:
Was able to fix my test user and seems to be working

Reply:
Yeah they're different. The `--color-panel-solid` seems to be white in light mode and a lighter black in dark mode whereas we want this to be light violet in light mode and a dark indigo in dark mode